doc/patch-author-guide.md

```diff
@@ -231,7 +238,7 @@ index 9bff3b28c357..18374fd35bd9 100644
 @@ -1751,6 +1751,8 @@ struct task_struct *fork_idle(int cpu)
  	return task;
  }
- 
+
```

Comment:
These whitespace changes are still scattered throughout the PR.  Is there a way to omit them so they don't add unrelated changes?

Reply:
It requires some manual work. But yeah, I can get rid of them.